cluster-configuration-example/cluster-configuration.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+# If corresponding values aren't be set in the machine list, the default value will be filled in.
+DefaultMachineProperties:
+  # Account with root permission
+  user: username
+  password: password
+  sshport: port
+
+
+MachineSku:
+
+  NC24R:
+    mem: 224
+    gpu:
+    # type: gpu{type}
+      type: teslak80
+      count: 4
+    cpu:
+      vcore: 24
+    #dataFolder: "/mnt"
+    #Note: Up to now, the only supported os version is Ubuntu16.04. Please do not change it here.
+    os: ubuntu16.04
+
+  D8SV3:
+    mem: 32
+    cpu:
+      vcore: 8
+    #dataFolder: "/mnt"
+    #Note: Up to now, the only supported os version is Ubuntu16.04. Pls don't change it here.
+    os: ubuntu16.04
+
+
+
+machinelist:
+
+    - hostip: IP
+      machinetype: D8SV3
+      etcdid: etcdid1
+      #sshport: PORT (Option)
+      #username: username (Option)
+      #password: password (Option)
+      k8s-role: master
```

Comment:
>k8s-role: master [](start = 6, length = 16)

Should we also defines k8s labels to "true"/"false" format? Such as k8s-master: true, k8s-worker: true.